http/http.go

```diff
@@ -184,6 +185,11 @@ func (h *HTTP) RunServer() (err error) {
 
 	log.Infof("Service listening on %s", GetConfig().ListenAddress)
 
+	// Since main blocks here until completion, tell systemd we're ready.
+	// This is a no-op if NOTIFY_SOCKET isn't set.
+	log.Debug("Notifying systemd of readiness")
```

Comment:
Instead of always showing this message you should check if we are actually running within systemd before showing this log and sending the notification. Maybe use something like `os.GetEnv("NOTIFY_SOCKET")` to check if it's set or not.

Reply:
Good point. Let me fix that.